src/seedsigner/views/seed_views.py

```diff
@@ -704,6 +707,9 @@ def run(self):
             selected_menu_num = 0
 
         else:
+            # imported locally instead of at module level to avoid flow-test warnings: "comparing int to MagicMock"
```

Comment:
This comment is too much detail for someone who's just interested in the main code. If the goal is to warn future flow-based test authors, adding a note in tests/README.md is probably a better place for it.

Maybe simplify to something like:
```
# imported locally to isolate hardware-dependent imports that cause problems in the test suite
```

Reply:
I agree that this comment is best left in the test modules.  The same code pattern appears in your original work, in my work, and in Nick's work, so a comment in the test module, and just the code pattern is best in the actual code.

Ty!

Reply:
The test readme is succinct and to-the-point, while I'm wordy.  

For now, I've opted to extend the comment in tests/base.py where test-coders are most likely to check should they find this class not successfully mocked out.

I'll participate in the test/readme documentation changes once others have discussed what is desired for flow test docs.